superset/assets/package.json

```diff
@@ -69,6 +70,7 @@
     "mapbox-gl": "^0.43.0",
     "mathjs": "^3.16.3",
     "moment": "2.18.1",
+    "mousetrap": "^1.6.1",
```

Comment:
Nice. we should sprinkle hotkeys everywhere! (and document them)

Reply:
Yeah, it's worth doing a PR introducing some basic shortcuts.

At some point I was playing with more shortcuts (left and right to step the scrubber, up and down to increase/decrease its width), but I was worried it could conflict with other things.